### PR TITLE
[FIX] FontAwesome: remove double click handler when detach

### PR DESCRIPTION
**Pull request by sebgeelen**

### CODE REVIEW

File: packages/plugin-odoo/src/OdooFontAwesomeDomObjectRenderer.ts [hunk 1]
```diff
@@ -13,16 +13,21 @@ export class OdooFontAwesomeDomObjectRenderer extends FontAwesomeDomObjectRender
         const domObject: DomObject = await super.render(node, worker);
         if (domObject && 'children' in domObject) {
             const fa = domObject.children[1];
+
+            const bdClickCallback = () => {
```
Comment (dmo-odoo) [inline packages/plugin-odoo/src/OdooFontAwesomeDomObjectRenderer.ts:17]: bdClick? ^^'

**Review by dmo-odoo — COMMENTED** [review]
No test to ensure this bug cannot come back ? :(

File: packages/plugin-odoo/src/OdooFontAwesomeDomObjectRenderer.ts (hunk 1 above)
Comment (sebgeelen) [reply packages/plugin-odoo/src/OdooFontAwesomeDomObjectRenderer.ts:17]: XD

**Comment by sebgeelen** [comment]
@dmo-odoo We don't yet have tests on odoo specific plugin.
To be honest, I don't even know how I should tests this on JW code only ...

So, should i spend time on this now ?

**Comment by dmo-odoo** [comment]
@sebgeelen I didn't realize this was the Odoo version of fontawesome renderer. It's fine sorry.

**Pull request merged**